src/models/SpecificationFile.ts

```diff
@@ -136,6 +137,7 @@ export async function load(filePathOrContextName?: string, loadType?: LoadType):
       return Specification.fromURL(filePathOrContextName);
     }
     await fileExists(filePathOrContextName);
+    if (command) {command.specFilePath = filePathOrContextName;}
```

Comment:
🤔 I have a bad feeling about injecting information on a foreign class like this. It doesn't feel elegant or "expected". Look at all the calls above like `await load(this, specFilePath)`. Or the signature of this method. It feels weird that we have to pass `this` or a `command` to load a file.

For instance, [this line](https://github.com/asyncapi/cli/pull/1304/files#diff-277d6f3a036a06f971ae75da2804a5b6b60cab44c03eb53d8d11a13ceef6e644R26) could become something like:

```js
this.specFile = await load(filePath);
this.specFilePath = this.specFile.getPath();
```

Reply:
Made the changes. 🙇